src/pudl/metadata/fields.py

```diff
@@ -4944,6 +5041,10 @@
         "type": "integer",
         "description": "Year the data was reported in, used for partitioning EPA CEMS.",
     },
+    "year_quarter": {
+        "type": "string",
+        "description": "Year quarter filing applies to.",
+    },
```

Comment:
I think for all other columns with this kind of information in them, we are using `report_date` -- sometimes with annual and other times with monthly frequency. In the FERC data I think it also describes quarterly data (although we only pull in Q4 information). Can we store a quarter using a date type, which conveys richer information about the contents, and can be used in conjunction with other report date data?

Reply:
Converted to `report_date`